src/Controls/samples/Controls.Sample/Pages/MainPage.cs

```diff
@@ -8,6 +8,18 @@ namespace Maui.Controls.Sample.Pages
 
 	public class MainPage : ContentPage, IPage
 	{
+		const string longLoremIpsum =
+			"Lorem ipsum dolor sit amet, consectetur adipiscing elit. " +
+			"Quisque ut dolor metus. Duis vel iaculis mauris, sit amet finibus mi. " +
+			"Etiam congue ornare risus, in facilisis libero tempor eget. " +
+			"Phasellus mattis mollis libero ut semper. In sit amet sapien odio. " +
+			"Sed interdum ullamcorper dui eu rutrum. Vestibulum non sagittis justo. " +
+			"Cras rutrum scelerisque elit, et porta est lobortis ac. " +
+			"Pellentesque eu ornare tortor. Sed bibendum a nisl at laoreet.";
```

Comment:
@hartez I cant actually use this long text in the vertical stack. As soon as the text lakes the label wider than the screen, it crashes on Android. Something about an index out of bounds in the measure.